manim/mobject/coordinate_systems.py

```diff
@@ -2103,30 +2105,29 @@ def add_coordinates(
         self.add(self.get_coordinate_labels(r_values, a_values))
         return self
 
-    def get_radian_label(self, number, stacked=True):
```

Comment:
Was `stacked` actually unused before?

Reply:
Yup, it only showed up in the definition of the method. Maybe @ralphieraccoon knows what it was supposed to be?

Reply:
I think it was originally supposed to be an argument that's now handled by `self.azimuth_compact_fraction` on creation. It looks like an orphan argument that was missed when it was merged, so feel free to delete it.